oar/core/advisory.py

```diff
@@ -749,3 +751,42 @@ def get_overall_grade(self):
         else:
             raise AdvisoryException(
                 f"error when accessing advisory containers - {self.errata_id}")
+
+    def check_kernel_tag(self):
+        """
+        Check kernel tag from advisory build image.
+
+        Returns:
+            bool: True if kernel is tagged with early-kernel-stop-ship.
+        """
+        #Check if impetus is image or not, if it's not image then skip this function.
+        if self.impetus != AD_IMPETUS_IMAGE:
+            logger.info("check kernel tag function skipped, only image advisory need to check.")
+            return False
+        #Get rhcos nvr from image advisory build
+        build_url = "https://errata.devel.redhat.com/api/v1/erratum/" + str(self.errata_id) + "/builds"
```

Comment:
you don't have to get build by your self, when advisory is initialized, `self.errata_builds` can be used to check

Reply:
There is some different between self.errata_builds and the return from HTTP request, I pasted the two responses in here: https://privatebin.corp.redhat.com/?5bbed92b72a641e8#6vJfR2HZLFBnVVfMDMoQTt3S9RnqpFGxDi4u7XcwKfg2

Reply:
is there any issue there, you just need to filter out the key starts with rhcos-x86_64

Reply:
ah, the response from different paths are different, unsigned builds are ignored in resp of `/advisory/$id/builds.json`, seems like we have to query the rhcos build with path `/api/v1/erratum/$id/builds`
```
def test_errata_builds(self):
    ad = Advisory(errata_id=145990, impetus=AD_IMPETUS_IMAGE)
    resp = ad._get(f"/api/v1/erratum/{ad.errata_id}/builds")
    name = None
    for v in resp.values():
        builds = v['builds']
        for b in builds:
            for k in b.keys():
                if k.startswith("rhcos-x86"):
                    name = k
                    break
    print(name)
```